### DISCUSSION TITLE
Official emergency assistance is required

### DISCUSSION THREAD

**Discussion by rubik-crypto** [filecoin-project/lotus#7744] [General Q&A]
Because of the fire
Lost all data
No backup score data
No metadata
Only the private key
How to take pledge collateral out
The total quantity is more than 100,000 FIL
Could you please help us

**Comment by jennijuju** [comment]
oof im so sorry to hear that. A couple of questions:
- whats your miner id
- whats the portion of cc sectors and deal sectors
   - if there are deals in the sectors,  is there any copy of the data/cid that we can work with the SP community to store them again so the data is not missing for the client.
- do you have any sealed or unsealed copy replicas.

however, this is not a lotus issue - transferring to the discussion in case you need more support from the community.

**Reply by rubik-crypto** [reply]
Thank you for your reply
One of the miner id is f023660
Only the private key
Everything else is lost

**Reply by jennijuju** [reply]
I noticed you (SP id: f023660) terminated 11,630 sectors on Dec 11, 2021.
 
On the same day, the following storage providers also terminated a large amount of sectors:
f023978, f023977, f023981, f023928, f023858, f023859, f023861, f023980, f023870, f024146, f024147, f023876, f023869, f023868, f023871, f024013, f024136, f024006, f024012, f024014, f024148, f024084, f023854, f024081, f023853, f024008, f024007, f023855, f024085, f023662, f023565.

Do you also own these other addresses as well? The locked collateral of f023660 alone is much less than the 100k FIL stated above, but maybe that is the locked collateral across all these addresses? If so, was their data also lost in the fire? Do you have any pictures/news about the fire?

Note - this is a great reminder to all Filecoin network participants of the importance of storing many redundant copies of data across globally distributed storage providers, so as to avoid any negative impacts from a single operation suffering an outage like this. All major storage operations must keep many copies of data at different locations around the world to be resilient to these sorts of problems and continue offering high quality storage services that keep client’s data secure & available. 


<img width="1040" alt="image" src="https://user-images.githubusercontent.com/42981373/145766351-3f276dc8-95ea-4d4f-a776-d2ced1e0419b.png">
